src/compositions/useDefaultSavedSearchFilter.ts

```diff
@@ -0,0 +1,44 @@
+import { useLocalStorage } from '@prefecthq/vue-compositions'
+import { ComputedRef, computed } from 'vue'
+import { getCacheKey, SavedSearchFilter, systemDefaultSavedSearch, isSameFilter } from '..'
+
+const customDefaultFlowRunsFilterKey = getCacheKey('prefect-ui-library-custom-default-flow-runs-filter')
+
+type UseDefaultSavedSearchFilter = {
+  value: ComputedRef<SavedSearchFilter>,
+  set: (value: SavedSearchFilter) => void,
+  remove: () => void,
+  isCustom: ComputedRef<boolean>,
+}
+
+const { value: custom, set: setCustom } = useLocalStorage<SavedSearchFilter>(customDefaultFlowRunsFilterKey)
+
+/**
+ * Flow runs filter specifically
+ * @returns
+ */
```

Comment:
Does this need to be updated or does @returns do something magically here

Reply:
oh oops. just a half-written doc string lmao don't mind me...